### ISSUE TITLE
Paperspigot world generation crash

### ISSUE THREAD

**Issue by AsVaidas** [PaperMC/Paper#1771]
### What behaviour is expected:
EWG needs to load the world as it should,

### What behaviour is observed:
Chunks can't load because of missing API (paperspigot is missing something that spigot has, and for that EWG cant work on paperspigot

### Steps/models to reproduce:
Load a huge world (in 50k x 50k world the issue appeard loading 10% of it)

### Plugin list:
EpicWorldGenerator (premium) https://www.spigotmc.org/resources/epicworldgenerator-1-8-8-1-13-2-support-all-update-aquatic-features.8067/

### Paper build number:
git-Paper-488 (MC: 1.13.2) (Implementing API version 1.13.2-R0.1-SNAPSHOT)

### Anything else:
Error:
[error.txt](https://github.com/PaperMC/Paper/files/2712709/error.txt)

**Comment by electronicboy** [comment]
As I stated on the forums, EWG is broken due to mapping changes in spigot which can only be fixed by the plugin author. Unless you can prove that it works on the latest version of spigot, this is not an issue with Paper.

Given that none of the staff team own the plugin, nobody on the staff team can really look into it, but we can mirror the very responses on the discussion forum for you: https://www.spigotmc.org/threads/epicworldgenerator-1-8-8-1-13-2-25-off-black-friday-sale-support-all-update-aquatic.71172/page-311#post-3254298

**Comment by AsVaidas** [comment]
Yea, 7 days of crashing with paperspigot (latest version) (10-30 times a day) and 20 days of perfect work (0 crashes in 20 days) with latest spigot.

That is why I have written "(paperspigot is missing something that spigot has, and for that EWG cant work on paperspigot)", I tested it on both, spigot and paperspigot

**Comment by electronicboy** [comment]
latest is not a version.

**Comment by AsVaidas** [comment]
This server is running CraftBukkit version git-Spigot-a1f2566-a4c555b (MC: 1.13.2) (Implementing API version 1.13.2-R0.1-SNAPSHOT)

**Comment by Spottedleaf** [comment]
That build is from the 23rd of Nov. Today is the 27th of December.

**Comment by electronicboy** [comment]
Your "latest" build of spigot is 50 versions out of date and is missing the changes on their end which broke EWG. https://spigot.club/checker#git-Spigot-a1f2566-a4c555b
ref "Mappings update"

**Comment by AsVaidas** [comment]
then why 1000 realese old spigot worked and the same time paper didn't, 500 realese old spigot worked and the same time paper didn't, 200 realese old spigot worked and the same time paper didn't, 50 realese old spigot worked and the same time paper didn't, and so on (because it wouldn't work now). So I think you get the point. Testing any version with paperspigot from 1.8 to 1.13.2, EWG just didn't work on paper, when spigot had no troble running it. You just can't put it on "Mappings update", cause I can bet, broken release of spigot would have no troble running EWG, while paper will. I have waited for 2 years to get a stable working EWG and I allways written the problem to EWG for not working, waiting for a fix. When I changed finally to spigot (what I don't want, really), the plugin just worked. And paperspigot realease was much newer then spigot, **because getbukkit doesn't update their version as fast, as my automated system, that updated paperspigot every server restart (or every night).**
